presto-main/src/main/java/com/facebook/presto/operator/OrderByOperator.java

```diff
@@ -147,8 +180,10 @@ public void finish()
         if (state == State.NEEDS_INPUT) {
             state = State.HAS_OUTPUT;
 
-            // sort the index
-            pageIndex.sort(sortChannels, sortOrder);
+            WorkProcessor<Page> resultPages = getSortedPages(
```

Comment:
Here we should also switch accounting from revokable to non-revokable as we don't support spilling when producing output (yet), but such allocation might fail then. We probably should do similar as in: https://github.com/prestodb/presto/pull/11654/files#r226286007